app/controllers/registers_controller.rb

```diff
@@ -7,14 +7,25 @@ class RegistersController < ApplicationController
 
   def index
     @search_term = search_term
-    @registers = Register.available
-                         .in_beta
-                         .search_registers(@search_term)
 
     # Redirect legacy URL to ensure we don't break anyone
     if params[:phase] == 'in progress'
       redirect_to registers_in_progress_path
     end
+
+    if params[:showby] == 'name' || @search_term.present?
+      @show_by_selected = 'name'
+      @registers_collection = Register.available
+                           .in_beta
+                           .search_registers(@search_term)
+                           .by_name
+    elsif params[:showby] == 'organisation'
+      @show_by_selected = 'organisation'
+      @registers_collection = Authority.with_a_register
+    else
```

Comment:
Should we hide the themes without a register, as we do for organisations? We are currently showing themes with nothing on them.

Reply:
Good point - I think that every theme/category _should_ have a register assigned to it, but that a assumption on my part.

Reply:
Done and pushed. Cheers!